### ISSUE TITLE
Re-Enable bytepushers.io on AWS

### ISSUE THREAD

**Issue by pouncilt** [byte-pushers/chucks-pick3#587]
BytePushers.io has expired.  We need to renew this domain name and ensure all the services still works.

**Comment by AmanAb428** [comment]
This domain is expired, we need to create AWS support ticket to restore the domain. Reference: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-restore-expired.html

As per the above blog we need to pay $133 as restoration fee for bytepushers.io domain. Reference: https://d32ze2gidvkk54.cloudfront.net/Amazon_Route_53_Domain_Registration_Pricing_20140731.pdf

@pouncilt shall I create AWS support ticket saying, we are agree to pay $133 as restoration fee?

**Comment by pouncilt** [comment]
Yes.  Please create AWS support ticket.  Yes we agree to pay $133 as a restoration fee.  Thank you.

**Comment by AmanAb428** [comment]
Created AWS support ticket with Case ID: 9583636401
https://console.aws.amazon.com/support/home?region=us-east-1#/case/?displayId=9583636401&language=en

**Comment by pouncilt** [comment]
Keep me posted.

**Comment by AmanAb428** [comment]
Got following reply from AWS team.
Hello there, 

Warm Greetings from AWS! 

I understand that you would like to restore the domain "bytepushers.io". 

We tried to restore the domain, however, **the payment method on Payer account was not validated yet,** due to which the restoration operation was not successful. 

I've created an outbound support case with subject line "Payment method for domain restoration" on Payer account with steps to validate the payment method. Kindly respond to this case after validating, we will surely try restoring the domain at the earliest. 

Have a nice day. 

We value your feedback. Please share your experience by rating this correspondence using the AWS Support Center link at the end of this correspondence. Each correspondence can also be rated by selecting the stars in top right corner of each correspondence within the AWS Support Center.

Best regards,
Meghana B.
Amazon Web Services

===============================================================

To share your experience or contact us again about this case, please return to the AWS Support Center using the following URL: [https://console.aws.amazon.com/support/home#/case/?displayId=9583636401&language=en ](https://console.aws.amazon.com/support/home#/case/?displayId=9583636401&language=en)

Note, this e-mail was sent from an address that cannot accept incoming e-mails.
To respond to this case, please follow the link above to respond from your AWS Support Center.

===============================================================

Don’t miss messages from AWS Support when you need help! Update your contact information:
[https://console.aws.amazon.com/billing/home#/account ](https://console.aws.amazon.com/billing/home#/account)

If you receive an error message when visiting the contact information page, visit:
[https://aws.amazon.com/premiumsupport/knowledge-center/iam-billing-access/ ](https://aws.amazon.com/premiumsupport/knowledge-center/iam-billing-access/)

AWS Support:
[https://aws.amazon.com/premiumsupport/knowledge-center/ ](https://aws.amazon.com/premiumsupport/knowledge-center/)

AWS Documentation:
[https://docs.aws.amazon.com/ ](https://docs.aws.amazon.com/)

AWS Cost Management:
[https://aws.amazon.com/aws-cost-management/ ](https://aws.amazon.com/aws-cost-management/)

AWS Training:
[http://aws.amazon.com/training/ ](http://aws.amazon.com/training/)

AWS Managed Services:
https://aws.amazon.com/managed-services/

**Comment by AmanAb428** [comment]
@pouncilt AWS is facing issue in making payment for the renewal of the domain, can you please check above comment?

**Comment by AmanAb428** [comment]
@pouncilt domain bytepushers.io is available in AWS Route 53. Please follow the steps from this document to register this domain: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html

**Comment by pouncilt** [comment]
@AmanAb428  bytepushers.io has been registered successfully.  Please ensure all the services are back up and document the process.
